web/package.json

```diff
@@ -12,7 +12,7 @@
     "start-console": "../scripts/start-console.sh",
     "i18n": "./i18n-scripts/build-i18n.sh && node ./i18n-scripts/set-english-defaults.js",
     "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
-    "lint": "eslint ./src ./integration-tests --fix &&  stylelint \"src/**/*.css\" --allow-empty-input --fix"
+    "lint": "eslint ./src ./integration-tests --fix &&  stylelint \"web/src/**/*.css\" --allow-empty-input --fix"
```

Comment:
Do we need the `web` path here? this commands should run from the `web` folder as they are listed on the package.json